src/lib.rs

```diff
@@ -175,6 +179,10 @@ impl HSpanContext {
     }
 }
 
+pub fn null_tracer() -> Tracer {
+    Tracer::new(NullSampler).0
+}
```

Comment:
What is the intention behind exposing the null tracer as part of the public API?

Reply:
https://github.com/holochain/lib3h/pull/494/files#diff-d70314a24a0d64aa9adbd6aac05d7de7R32

Reply:
@ddd-mtl I object to the use of a lazy_static Mutex<Tracer> there, but that is another issue in another repo :) I think the thing using the Tracer, in this case GhostEndpointFull, should own the Tracer directly. And then there is no need for a null "default" tracer just to satisfy the static Mutex.

As far as this repo is concerned, I guess it's not that big of a deal, but that's exactly the kind of pattern I don't want to encourage, which is why I would still recommend not exposing this publicly. If it were meant to be used this way, Tracer would have a Default impl which is this function.